rules/no-implicit-end.js

```diff
@@ -63,6 +63,10 @@ module.exports = function() {
       return false;
     }
 
+    if (is(node, 'bpmn:Activity') && is(node.$parent, 'bpmn:AdHocSubProcess')) {
+      return false;
+    }
+
     if (is(node, 'bpmn:EndEvent')) {
       return false;
```

Comment:
It is misleading that this is false, cf. https://github.com/bpmn-io/bpmnlint/pull/181#issuecomment-2739396363.